### PR TITLE
Simplify the README

### PR DESCRIPTION
**Pull request by guibou**
I want it to be really simple.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,32 +9,3 @@ When designing a rule, we want to follow some guidelines:
...
-- Error
-
-An *Error* is an assumption on the codebase which should never be violated. In the default usage,
-those are meant to fail a build (in a CI system).
-A *Warning* is an assumption on the codebase that is important to report but which is not deemed
-critical enough to block the build.
-An *Info* is simply a notification for the user that can give some context about a comment check.
-
-The UI will provide flags to ignore some notifications or change their level (e.g. consider all
-warnings as fatal or the converse).
```
Comment (guibou) [inline CONTRIBUTING.md:30]: Removing that one is perhaps too much.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -9,32 +9,3 @@ When designing a rule, we want to follow some guidelines:
...
-
-### Performance
-
-Parsing of text file and evaluating of rules (especially the ones which need Internet access) should
-be done in a parallel way to reduce the time used by the executable.
-
```
Comment (guibou) [inline CONTRIBUTING.md:36]: We don't really care now, that's noise.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -9,32 +9,3 @@ When designing a rule, we want to follow some guidelines:
...
-## Hacking
-
-Krank is written in Haskell, because why not ;) It aims at being available in Hackage / Stackage
-with minimal (if zero) system dependencies.
```
Comment (guibou) [inline CONTRIBUTING.md:40]: Again, that's noise. People will see that's it is written in Haskell, and the note about Hackage / Stackage and few dependencies will brings nothing except false assumptions. If someone want to contribute something extra with a huge dependency, let him contribute, we'll filter by PR.

File: HACKING.md [hunk 4]
```diff
@@ -24,4 +24,9 @@ Should work.
 
 # Hacking
 
-Edit code, ensure that test are passing and open a pull request on github. Thank you ;)
+Edit code, ensure that test are passing and open a pull request on Github. Thank you ;)
+```
+
+You will get an output similar to:
+
+```
```
Comment (guibou) [inline HACKING.md]: This is a leftover, I should remove it!

File: README.md [hunk 5]
```diff
@@ -1,48 +1,44 @@
...
-Krank will parse the comments in the code and check fact, so the code won't bitrot. For example, it
-will:
-
-- look for link to github issue / pull request and warn you if there are now closed: #1
-- check for cross reference in comments: #2
-- check for outdated / bitroting TODOs: #3, #4
-- check for association between code and comment: #5
-
-Check the github issues section to see the list of possible checkers.
+Comments are part of our code and are not usually tested correctly. Hence their content can become incoherent or obsolete. Krank tries to avoid that by running checkers on the comment themselves.
```
Comment (guibou) [inline README.md:5]: I removed the list of things that krank could do, that's science fiction for now and does not belongs to the readme.

File: HACKING.md (hunk 4 above)
Comment (guibou) [reply HACKING.md]: Fixed.

File: CONTRIBUTING.md (hunk 1 above)
Comment (gbataille) [reply CONTRIBUTING.md:30]: this is redundant with the "user doc" so I'm ok with that

**Review by gbataille — APPROVED** [review]

**Pull request merged**